.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,10 @@ Closes issue(s):
 
 Need for Final Call:
 << choose one of the following and remove the rest >>
+<< leave the "How to Review and Approve" section in your pull request description >>
```

Comment:
Seems weird to have this as one of the "Need for Final Call" options.

Reply:
Indeed. Moved it to the instructions in the PR Etiquette section